policyengine_us/variables/gov/states/ne/tax/income/credits/ctc/refundable/ne_refundable_ctc_eligible_child.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_us.model_api import *
+
+
+class ne_refundable_ctc_eligible_child(Variable):
+    value_type = bool
+    entity = Person
+    label = "Nebraska refundable Child Tax Credit eligible child"
+    definition_period = YEAR
+    reference = (
+        "https://revenue.nebraska.gov/about/2023-nebraska-legislative-changes"
+    )
+    defined_for = StateCode.NE
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.ne.tax.income.credits.ctc.refundable
+        is_dependent = person("is_tax_unit_dependent", period)
+        age = person("age", period)
+        age_eligible = age <= p.age_threshold
+        has_childcare_expenses = (
```

Comment:
```suggestion
        # We assume that the child is enrolled in a child care program 
        # licensed pursuant to the Child Care Licensing Act
        has_childcare_expenses = (
```

Reply:
Create separate variables for the childcare criteria to reflect the law. Fill them in with imputations if appropriate, though seems like it'll be a subset of those with pre-subsidy childcare expenses.

Reply:
> Create separate variables for the childcare criteria to reflect the law. Fill them in with imputations if appropriate, though seems like it'll be a subset of those with pre-subsidy childcare expenses.

just want to confirm, as we previously missed that the legal code actually includes an "or" and not an "and", should I have three separate variables for the 1a-c conditions